### PR TITLE
Feat(exporter-js): Introduce JS exporter from Supernova #DS-1013

### PR DESCRIPTION
**Pull request by crishpeen**
### Issue reference

---

### Before submitting the PR, please make sure you do the following

- [ ] Read the [Contributing Guidelines](https://github.com/lmc-eu/spirit-design-system/blob/main/CONTRIBUTING.md).
- [ ] Follow the [PR Title/Commit Message Convention](https://github.com/lmc-eu/spirit-design-system/blob/main/CONTRIBUTING.md#commit-conventions).
- [ ] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

File: exporters/js/exporter.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "id": "eu.lmc.exporter-spirit-js",
+  "name": "Spirit JS Exporter",
+  "description": "Spirit JS Exporter",
+  "author": "Jan Kryšpín <jan.kryspin@lmc.eu>",
```
Comment (dlouhak) [inline exporters/js/exporter.json:5]: Like a pro

**Comment by literat** [comment]
It generates
```js
export const bodyLargeButton = {
    mobile: {
        fontFamily: "'Inter', sans-serif",
        fontSize: '1.125rem',
        fontStyle: 'normal',
        fontWeight: NaN,
        lineHeight: 1.5,
    },
};
```
But this is not in SCSS-generated tokens.

**Comment by literat** [comment]
`gradients.ts` has been generated empty.

**Comment by literat** [comment]
> Good Job 👍 A lot of code and big effort from you and AI ;-)
> 
> I have tried to run it a examine the generated files manually and found some inconsistencies with SCSS-generated tokens. Please see my comments in this PR.

Ok, badly set configuration on my side.

**Pull request merged**